src/runtime_src/core/include/xrt/xrt_aie.h

```diff
@@ -163,6 +164,62 @@ class bo : public xrt::bo
   }
 };
 
+class profiling_impl;
+class profiling : public detail::pimpl<profiling_impl>
+{
+public:
+  enum class profiling_option : int { io_total_stream_running_to_idle_cycles = 0, io_stream_start_to_bytes_transferred_cycles = 1, io_stream_start_difference_cycles = 2, io_stream_running_event_count = 3 };
```

Comment:
```suggestion
  enum class profiling_option : int 
  { 
    io_total_stream_running_to_idle_cycles = 0, 
    io_stream_start_to_bytes_transferred_cycles = 1, 
    io_stream_start_difference_cycles = 2, 
    io_stream_running_event_count = 3 
  };
```
You would maybe want to document these?

```
/**
 * @enum profiliing_options - ...
 *
 * @var enumerator 
 *    What it does
 * /
```
 

I know the enumerators are long, so maybe they explain what needs to be explained?